libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -86,15 +87,20 @@ - (void)setUp {
     // Delete the content of the global library directory
     NSString *globalLibraryDirectory = [[SFDirectoryManager sharedManager] directoryForUser:nil type:NSLibraryDirectory components:nil];
     [[[NSFileManager alloc] init] removeItemAtPath:globalLibraryDirectory error:nil];
-
     // Set the oauth client ID after deleting the content of the global library directory
     // to ensure the SFUserAccountManager sharedInstance loads from an empty directory
     self.uam = [SFUserAccountManager sharedInstance];
-    self.uam.oauthClientId = @"fakeClientIdForTesting";
+    [SFAuthenticationManager sharedManager].oauthClientId = @"fakeClientIdForTesting";
 
     // Ensure the user account manager doesn't contain any account
+    NSArray *userAccounts = [[SFUserAccountManager sharedInstance] allUserAccounts];
```

Comment:
I think that's fine.  I'm almost certain it was specifically added as a unit testing helper.

I don't know of any specific issues that would arise from changing the implementation to address persisted accounts as well, but if there are, I think we should just otherwise work through them, and change it anyway.  Particularly now that we have an in-memory persister object that we could leverage in these test scenarios, as necessary.